### ISSUE TITLE
'destroy' command

### ISSUE THREAD

**Issue by ehelms** [lago-project/lago#110]
I'd like to request a destroy command that properly stops, does any cleanup and removes an environment. I've already ran into accidentally doing `rm -rf environment/` without running `lago stop`. Help me protect myself from myself.

**Comment by mykaul** [comment]
cleanup is not good enough?

**Comment by ehelms** [comment]
I don't really understand what cleanup is doing. If I name my prefix 'environment', I have to:

```
 cd environment/
lago cleanup
cd ..
rm -rf environment/
lago init environment foo.json
```

When, as a user, it would be nice to do:

```
lago destroy environment
lago init environment foo.json
```

**Comment by mykaul** [comment]
I don't bother to rm the environment, but yes, I do perform 'cd
<environment> && lagocli cleanup && cd -' which is a bit annoying.

**Comment by david-caro** [comment]
This might be solved along with #109 (having the .lago directory as the prefix allows to easily remove it from command line and run commands without having to actually be inside it, just by being in a sibling dir, similar to git)
